.travis.yml

```diff
@@ -1,15 +1,23 @@
 language: java
 env:
-  - VERIFY_USE_PUBLIC_BINARIES=true
+  global:
+    - VERIFY_USE_PUBLIC_BINARIES=true
+    - secure: "Gj2/14PjUQrl8cYNCFcsLTuNNQAdhcKs+X5H3roaUsN29ozfyApoAhtG6oi2jfiB3md2cTVBvWWi+OFYS5paW+uM2e7VHCDpKsusK/lhVK9jMELgqS0eE9RcMDe3BoD/UZzKzF2K8vl/yhKXV024kW5FWgP/RevKFke+cwuJf2QXQ+nMAYvF0RuTyZFw1K0Lg5mAgnUMoHi6XiELMrzBCOTmb+yp5Pz+m6bgKkjDxN8Yds3U4h7FapjkhN6+wA03CFoLDEU0mecRaVusK/djlwVXBGYPDj+l/j4C0xnUZIkHZCr15823T3w6CrZabGfmNoNjwYO9ruYZJWM93i6ojZxz6On8AVkUo8Onp9PAzV8Nh7EyN+7AQ0lAXFeN71J/+6eD6F1e/UXuNEd/c7OEue72tBJZ6yvDtI4iQyZ4S5SLHQlVZGDd5n1gTn1FaVgiFt2g+OJASgpnnSHF9gv7FCbDU/9pzlJXbiMSVrIaSpVr/hDaoZOKzLyWYBgMAMFLCeG80fjhDYLjft7qMnnFN51QdVkyrvbsE8Ouh7znxqkpBEywsqdWZjHZz3+KD6NlgMU2iTSdGhrqGAlQMI2kSmwkirch+qbQHWw3dH2i9trsbGNQTSQUEtgqNsqblQPcpUmg0ISrbNZRuLviFiHAtoLtaN6815kvzl8WZotrB/4="
+    - secure: "GnVoNGoi0tpNVORu/xH6iL4pGFTBtk3lxFyBSn/nrdSzNKa8VFdwcItLrNMAqDPQ0J9Mj5LhJfciRFX012uEIb+AJYYEIPMnKV+vYbjceagj9lM5X/NXvPWuS6BByE6sbV+hxGA/iSbxInIplaA2YS8HyuB1UesD12u35YXgt4sSS4i3xiLW90NZ6poY7xXiYM2ghY9dLCXqn6DggCVR+4Jm77eYvJN89Tk3bU7fnHbOMz6yOnEADUG84AtMBfEoQxuelIc2dJ6pto5A/Am6s4LKsWnSCMZsBgZ3VpIdM26a+lg68CvBIe4aHDcQc6daAVvpfcVVG3XXhNEKyHYskLFeCPxIgpXvOXTMtpPn79Dw0nSviPbznp3pQ7Jqx11ztjYLj4s5rE5AuJtD5xF8SxmVpgmQZhAMC3VdcVZ5EWkGGnno19Y2h3vU8T4btD3nqQ0eG5dHFKUPV5ifHE8u+Z0gYBVgmNPp5z5QC98bTjSubE8HnSRJBX+wpjyCyKZSjArByDzHGLKqEMMvj4GBsrAh9QcTDBPpyH6yF3HxSf1eunF8h3NAeSwVkAUrStUtthPLP1NEKc5qB7yEbBCXu5eowMsbpZ3au5hSeNVoOOJnBoOk9YDJ7i3ob9kU5c2ABBTIzNsWuVmc9O/psLUJYs6fX+u4qmSil4GDehICpxY="
 jdk:
   - oraclejdk8
   - openjdk8
   - openjdk11
+before_install:
+  - sudo apt-get install jq
+  - curl -u ida-codacy-bot:$GITHUB_PAT -LSs $(curl -u ida-codacy-bot:$GITHUB_PAT -LSs https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r '.assets | map({content_type, browser_download_url} | select(.content_type | contains("java-archive"))) | .[0].browser_download_url') -o codacy-coverage-reporter-assembly.jar
```

Comment:
Do we the `-u ida-codacy-bot:$GITHUB_PAT` authentication parms? The repos queried seem to be publicly accessible.

Reply:
Ignore me ... just re-read the PR description! 🙄